src/pages/graphql/schema/b2b/requisition-list/mutations/delete-items.md

```diff
@@ -0,0 +1,91 @@
+---
+title: deleteRequisitionListItems mutation
+edition: b2b
+contributor_name: EY
+---
+
+# deleteRequisitionListItems mutation
+
+The `deleteRequisitionListItems` mutation removes items from the specified requisiton list for the logged in customer.
+
+This mutation requires a valid [customer authentication token](../../../customer/mutations/generate-token.md).
```

Comment:
This link doesn't render. I don't understand why.

Reply:
Weird... it render ok for me.

![Screen Shot 2022-11-21 at 4 31 29 PM](https://user-images.githubusercontent.com/13662379/203171287-2337155e-2bec-4252-a9ba-c2b12f6d98b8.png)

Reply:
The error was in https://developer-stage.adobe.com/commerce/webapi/graphql/schema/b2b/requisition-list/mutations/delete/. Fixed in https://github.com/AdobeDocs/commerce-webapi/pull/57/commits/a42d9fb0d9a8a643d0e6a88e6e74129dc89a9022.